code/modules/parallax/parallax_controller.dm

```diff
@@ -187,8 +159,8 @@ var/global/parallax_enabled = TRUE
 /mob/proc/update_parallax(datum/component/component, turf/old_turf, turf/new_turf)
 	src.client?.parallax_controller?.update_parallax_layers(old_turf, new_turf)
 
-/mob/proc/update_parallax_z()
-	src.client?.parallax_controller?.update_parallax_z()
+/mob/proc/update_parallax_z(datum/component/component, old_z_level, new_z_level)
```

Comment:
any arbitrary component?

Reply:
No, specifically `/datum/component/complexsignal/outermost_movable`, however the `component` argument here will never be used be parallax code, however it will still be passed by the complex signal, therefore we must accommodate for it.